test/belongs-to.test.js

```diff
@@ -0,0 +1,52 @@
+import test from 'ava';
+
+import BemEntityName from '../index';
+
+test('should resolve belonging between elem and block', t => {
+    const entity1 = new BemEntityName({ block: 'block' });
+    const entity2 = new BemEntityName({ block: 'block', elem: 'elem' });
+
+    t.true(entity2.belongsTo(entity1));
+    t.false(entity1.belongsTo(entity2));
+});
+
+test('should not detect belonging between two block', t => {
+    const entity1 = new BemEntityName({ block: 'block2' });
+    const entity2 = new BemEntityName({ block: 'block' });
+
+    t.false(entity1.belongsTo(entity2));
+    t.false(entity2.belongsTo(entity1));
+});
+
+test('should resolve belonging between block and its mod', t => {
+    const entity1 = new BemEntityName({ block: 'block' });
+    const entity2 = new BemEntityName({ block: 'block', modName: 'mod', modVal: 'key' });
```

Comment:
Let's use `mod: { name, val }` field instead of `modName` + `modVal` in tests and docs.